apps/wing-console/console/ui/src/services/use-map.ts

```diff
@@ -228,6 +228,21 @@ export const useMap = ({ expandedItems }: UseMapOptions) => {
     return bridgeConnections({
       connections:
         rawConnections
+          .map((connection) => {
```

Comment:
@eladb this is a token connection, right? (from somewhere to the Api->url)

Reply:
ahhhh, it is from the container node to the `url`...
looks strange

Reply:
![Screenshot 2024-06-06 at 16 49 35](https://github.com/winglang/wing/assets/2538825/a56f893f-2b2d-455f-afc7-06b231692209)

Reply:
It's a scenario we should think about. The token is hidden so it's connections go to the parent, so instead of ApiUsersService/Token->ApiUsersService/Api we have ApiUsersService->ApiUsersService/Api which effectively means a parent connecting to its child.

Not sure if that makes sense or not, although would discuss outside of this issue maybe?